grpc/src/test/java/com/linecorp/armeria/client/grpc/GrpcClientTest.java

```diff
@@ -305,6 +310,63 @@ void contextCaptorAsync() {
         }
     }
 
+    @Test
+    void callOptionsExecutorIsRespected() {
+        final ExecutorService executor = Executors.newSingleThreadExecutor(
+                ThreadFactories.builder("my-executor")
```

Comment:
nit: Could we use a name that's more distinguishable? I imagine someone else write a similar test case in some other place with the same thread name prefix.